package.json

```diff
@@ -26,12 +26,5 @@
   },
   "devDependencies": {
     "coffeelint": "^1.10.1"
-  },
-  "activationCommands": {
-    "atom-workspace": [
-      "sync-settings:backup",
-      "sync-settings:restore",
-      "sync-settings:view-backup"
-    ]
```

Comment:
Yes, according to https://atom.io/docs/latest/hacking-atom-package-word-count#package-json if no activationCommands are specified the package is activated on atom start, otherwise it waits for one of the commands to be executed.